cronback-lib/config.rs

```diff
@@ -20,6 +20,7 @@ pub enum Role {
     Api,
     Dispatcher,
     Scheduler,
+    ProjectStore,
```

Comment:
Store suffix here might be confusing. We use this convention to define database interface, so maybe something like `ProjectSrv` might be a less confusing option?

I'm planning to follow up on this and rename the rest of configs for services to follow the same convention — `/s/Api/ApiSrv/`, and etc.